### PR TITLE
Update create-clusert-database-cli.md

### PR DESCRIPTION
**Pull request by AssafTaubenfeldMSFT**
We released new CLI client and the command has been changed a bit. I have updated the docs to match the new syntax.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@AssafTaubenfeldMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/create-cluster-database-cli.md [hunk 1]
```diff
@@ -50,14 +50,15 @@ The following steps are not required if you're running commands in Azure Cloud S
...
     ```azurecli-interactive
-    az kusto cluster create --name azureclitest --sku D11_v2 --resource-group testrg
+    az kusto cluster create --name azureclitest --sku name="Standard_D13_v2" tier="Standard" --resource-group testrg --location westus
     ```
 
    |**Setting** | **Suggested value** | **Field description**|
    |---|---|---|
    | name | *azureclitest* | The desired name of your cluster.|
-   | sku | *D13_v2* | The SKU that will be used for your cluster. |
+   | sku | *Standard_D13_v2* | The SKU that will be used for your cluster. Parameters: name- the name of the sku. tier- the teir of the sku. |
```
Comment (radennis) [inline data-explorer/create-cluster-database-cli.md:59]: typo

**Comment by PRMerger9** [comment]
@AssafTaubenfeldMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/create-cluster-database-cli.md [hunk 2]
```diff
@@ -50,14 +50,15 @@ The following steps are not required if you're running commands in Azure Cloud S
...
     ```azurecli-interactive
-    az kusto cluster create --name azureclitest --sku D11_v2 --resource-group testrg
+    az kusto cluster create --name azureclitest --sku name="Standard_D13_v2" tier="Standard" --resource-group testrg --location westus
     ```
 
    |**Setting** | **Suggested value** | **Field description**|
    |---|---|---|
    | name | *azureclitest* | The desired name of your cluster.|
-   | sku | *D13_v2* | The SKU that will be used for your cluster. |
+   | sku | *Standard_D13_v2* | The SKU that will be used for your cluster. Parameters: name- the name of the sku. tier- the tier of the sku. |
```
Comment (orspod) [inline data-explorer/create-cluster-database-cli.md:59]:
```suggestion
   | sku | *Standard_D13_v2* | The SKU that will be used for your cluster. Parameters: *name* -  The SKU name. *tier* - The SKU tier. |
```

File: data-explorer/create-cluster-database-cli.md [hunk 3]
```diff
@@ -74,16 +75,15 @@ If the result contains `provisioningState` with the `Succeeded` value, then the
...
    |---|---|---|
    | cluster-name | *azureclitest* | The name of your cluster where the database will be created.|
-   | name | *clidatabase* | The name of your database.|
+   | database-name | *clidatabase* | The name of your database.|
    | resource-group | *testrg* | The resource group name where the cluster will be created. |
-   | soft-delete-period | *P365D* | Signifies the amount of time that data will be kept available to query. See [retention policy](kusto/management/retentionpolicy.md) for more information. |
-   | hot-cache-period | *P31D* | Signifies the amount of time that data will be kept in cache. See [cache policy](kusto/management/cachepolicy.md) for more information. |
+   | read-write-database | *P365D* *P31D* *westus* | The type of the database. Parameters: soft-delete-period- Signifies the amount of time that data will be kept available to query. See [retention policy](kusto/management/retentionpolicy.md) for more information. hot-cache-period- Signifies the amount of time that data will be kept in cache. See [cache policy](kusto/management/cachepolicy.md) for more information. location: westus-The location where the database will be created. |
```
Comment (orspod) [inline data-explorer/create-cluster-database-cli.md:86]:
```suggestion
   | read-write-database | *P365D* *P31D* *westus* | The database type. Parameters: *soft-delete-period* - Signifies the amount of time the data will be kept available to query. See [retention policy](kusto/management/retentionpolicy.md) for more information. *hot-cache-period* - Signifies the amount of time the data will be kept in cache. See [cache policy](kusto/management/cachepolicy.md) for more information. *location* -The location where the database will be created. |
```

**Comment by PRMerger10** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**